src/cosmicqc/analyze.py

```diff
@@ -48,6 +47,12 @@ def identify_outliers(
         feature_thresholds_file: Optional[str] = DEFAULT_QC_THRESHOLD_FILE,
             An optional feature thresholds file where thresholds may be
             defined within a file.
+        include_threshold_scores: bool
+            Whether to include the threshold scores in addition to whether
+            the threshold set passes per row.
+        export_path: Optional[str] = None
+            An optional path to export the data using SCDataFrame export
+            capabilities. If None no export is performed.
```

Comment:
Should you include here what file types are supported for exporting the data?

Reply:
Good call - I'll update the docs here, thanks!